Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ruby
+
+RUN gem install --no-document etcenv
+
+RUN groupadd -g 500 core && useradd -u 500 -g 500 core
```

Comment:
Ah, right, had a specific use case for that. But that's not even the case anymore. I'll remove that commit.